chart/kubeapps/templates/apprepository-rbac.yaml

```diff
@@ -32,6 +32,18 @@ rules:
       - jobs
     verbs:
       - create
+---
+# Kubeapps can read and watch its own AppRepository resources cluster-wide.
```

Comment:
shouldn't we enable this behind a feature flag for the moment? that way we avoid "breaking" changes in the chart while the feature is being developed

Reply:
I can put it behind a flag, but didn't see much point given that this doesn't affect existing behaviour, just enables the features which will themselves be behind a feature flag to work properly. Let me update.

Reply:
> given that this doesn't affect existing behaviour

well it affects in the sense that we are now installing a cluster wide resource (apart than the CRD)

Reply:
PTAL :)